classes/book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
- [ ] Kindly make sure to add a method for rentals. As it is required.

Reply:
> * [x]  Kindly make sure to add a method for rentals. As it is required.

Oh, right. Thanks